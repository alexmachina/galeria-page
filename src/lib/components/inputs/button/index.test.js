import React from "react";
import renderer from "react-test-renderer";
import Button from "./";

describe("Components", () => {
  describe("Header", () => {
    it("Renders correctly", () => {
      const tree = renderer
        .create(<Button title="Magic" onClick={a => a} />)
        .toJSON();

      expect(tree).toMatchSnapshot();
    });
  });
});
