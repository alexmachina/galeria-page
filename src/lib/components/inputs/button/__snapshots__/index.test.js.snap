// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Header Renders correctly 1`] = `
<button
  className="root-0-2-1"
  onClick={[Function]}
  type="submit"
>
  Magic
</button>
`;
